### PR TITLE
 New v63003/gtm8789 subtest (Tests GTM-8789 in V63003)

### PR DESCRIPTION
**Pull request by VinayYottaDB**
Tests the following release note:

The Update Process operates correctly when a trigger issues a SET $ZGBLDIR to switch the current global directory (without doing a NEW $ZGBLDIR). Previously it used to switch the global directory correctly the first time the trigger was invoked but later invocations of the trigger did not switch the global directory resulting in any updates in the trigger code unintentionally happening in the primary global directory with which the update process was started. A workaround in prior versions was to do a NEW $ZGBLDIR before the SET $ZGBLDIR in the trigger code.

### CODE REVIEW

File: v63003/instream.csh [hunk 1]
```diff
@@ -47,6 +47,7 @@
 # gtm8801	    [vinay] Tests ^%YGBLSTAT works on a cmake build
 # gtm8842	    [vinay] Tests TRIGGER_MOD appropriately restricts ZBREAK and ZSTEP
 # gtm8858	    [vinay] Demonstrates the improved available information in cases of apparent database integrity issues
+# ydb293	    [vinay] Tests the update process operates correctly when triggering SET $ZGBLDIR
```
Comment (nars1) [inline v63003/instream.csh]: when triggering -> with triggers and

Comment (nars1) [reply v63003/instream.csh]: The ydb293 subtest should be part of r124 test (not v63003 test).

File: r124/u_inref/ydb293.csh [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+echo ""
+$MSR RUN INST1 '$ydb_dist/mumps -run ydb293'
+
+
+# Dump globals on primary and secondary side
+echo "Dumping globals on primary side"; echo "--------------------------------"; $ydb_dist/mumps -run dump^ydb293
+sleep 1 # to ensure stuff is replicated across to secondary
+echo "Dumping globals on secondary side"; echo "--------------------------------"; $sec_shell "$sec_getenv; cd $SEC_SIDE; $ydb_dist/mumps -run dump^ydb293"
+echo $! >& pid.txt
```
Comment (nars1) [inline r124/u_inref/ydb293.csh]: What do lines 46 and 47 do?

Comment (VinayYottaDB) [reply r124/u_inref/ydb293.csh]: oops forgot to remove them

**Pull request merged**